python/requirements/tune/requirements_tune.txt

```
@@ -46,7 +46,14 @@ torch==1.8.1+cpu;sys_platform!="darwin"
 torchvision==0.9.1+cpu;sys_platform!="darwin"
 
 
-transformers==4.8.1
+transformers==4.9.1
 wandb==0.10.29
 xgboost==1.3.3
 zoopt==0.4.1
+
+# Dependencies for Hugging Face examples:
+# `python/ray/util/sgd/v2/examples/transformers/transformers_example.py`
+accelerate
```

Can you pin this to the latest version?

```suggestion
accelerate==0.3.0
```